Ginger/GingerCoreNET/ALMLib/ALMUserConfig.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Amdocs.Ginger.Repository;
+using GingerCore;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GingerCoreNET.ALMLib
+{
+    public class ALMUserConfig : RepositoryItemBase
+    {
+        private string mALMUserName;
+        [IsSerializedForLocalRepository]
+        public string ALMUserName
+        {
+            get { return mALMUserName; }
+            set
+            {
+                if (mALMUserName != value)
+                {
+                    mALMUserName = value;
+                    OnPropertyChanged(nameof(ALMUserName));
+                }
+            }
+        }
+
+        [IsSerializedForLocalRepository]
+        public string EncryptedALMPassword { get; set; }
+        public string ALMPassword
+        {
+            get
+            {
+                bool res = false;
+                string pass = EncryptionHandler.DecryptString(EncryptedALMPassword, ref res);
+                if (res && String.IsNullOrEmpty(pass) == false)
+                    return pass;
+                else
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'else' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)